tools/run_tests/run_xds_tests.py

```diff
@@ -828,12 +828,38 @@ def test_path_matching(gcp, original_backend_service, instance_group,
                 {
                     "UnaryCall": alternate_backend_instances,
                     "EmptyCall": original_backend_instances
+                }),
+            (
+                # This test case is similar to the one above, but with route
+                # services swapped, has two routes (full_path and the default)
+                # to match EmptyCall, and refer to the same service.
```

Comment:
Nit: I don't understand what "refer to the same service" means here

Reply:
Comment updated.